public/locales/en/panel-navigation.json

```diff
@@ -0,0 +1,52 @@
+{
+  "hidden-label-focus": "Focus mode",
+  "hidden-label-anchor-aim": "Anchor & Aim mode",
+  "anchor-aim": {
+    "anchor-aim-view": {
+      "error": {
+        "no-anchor-aim-property": "Anchor or aim property property does not exist"
+      },
+      "title": "Anchor / Aim",
+      "info": {
+        "about-icons": "Set an anchor (<anchorIcon/>) that the camera attaches to, and an aim (<aimIcon/>) for the camera to look at. The aim will be kept in the same position as time changes.",
+        "shift-keybind": "When selecting the aim/anchor, the chosen node will be targeted to be centered in the view. Hold <keybind>Shift</keybind> on-click to set anchor/aim without targeting."
+      },
+      "retarget-anchor-tooltip": "Retarget anchor",
+      "retarget-aim-tooltip": "Retarget aim",
+      "retarget-aim-tooltip-no-aim": "No aim is set",
+      "no-aim": "No aim",
+      "filter-list-placeholder": "Search for a new anchor/aim..."
+    },
+    "anchor-aim-list-entry": {
+      "anchor-tooltip": "Set and target anchor",
+      "anchor-aria-label": "Set anchor to",
+      "aim-tooltip": "Set and target aim",
+      "aim-aria-label": "Set aim to"
+    },
+    "focus-view": {
+      "title": "Focus",
+      "info":  {
+        "click-focus-button": "Click the <focusIcon /> button to focus/retarget object.",
+        "shift-keybind": "Hold <keybind>Shift</keybind> when clicking to set as focus/anchor without retargeting.",
+        "ctrl-keybind": "Hold <keybind>Ctrl</keybind> when clicking to keep current camera velocities."
+      },
+      "filter-list-placeholder": "Search for a new focus..."
+    }
+  },
+  "menu-button": {
+    "no-anchor": "No anchor",
+    "focus": "Focus",
+    "anchor": "Anchor",
+    "aim": "Aim"
+  },
+  "navigation-settings": {
+    "title": "Navigation Settings",
+    "include-non-focusable-label": "Include Non-focusable Nodes in Search",
```

Comment:
```suggestion
    "include-non-focusable-label": "Include non-focusable Nodes in Search",
```

Reply:
Looking at the capitilization of the rest of this string, I don't agree with this. 

We could de-capitalize the entire label, but then it would mismatch with the property inputs in this menu: 
![image](https://github.com/user-attachments/assets/0c68fac0-178f-4d16-9956-343b7f62c758)

Reply:
Ah.. it's a discrepancy with AP style (which I'm more familiar with) and APA style (which we decided to use).  AP Style would be Non-Focusable or non-focusable, but APA style is Non-focusable.

Reply:
So, the action is?

Reply:
Action is disregarding my comment and staying with Non-focusable